schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-1

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
@istreeter doing a diff with 1-0-0 a lot of these fields now have a `maximum` as well as a few now allowing `object, null` pairing over just `object`.  Just want to confirm that the addition of new limits and type groupings is not going to cause issues in any of our loaders / data-warehouses?

I am not entirely certain if this schema gets tracked into pipelines to be fair but if it was would these changes get automatically represented in their respective columns?

Reply:
The question is still valid, just wanted to say that the schema is not being used in pipelines, it is just to help developers better define their configurations (some IDEs may use it for validating the JSONs).

Reply:
I tested this on a pipeline with a loader.  Even if it's not expected to be used in tracking, it was worth testing.

It is completely safe, because the field `configurationBundle` is a complex object, so RDB transformer simply stringifies the whole field, and RDB Loader loads it as a string.  Any change you make to a sub-field (e.g. `configurationBundle.namespace`) does not bother the transformer/loader at all.